operation_test.go

```diff
@@ -389,12 +389,16 @@ func TestParseParamCommentBodyArray(t *testing.T) {
 	expected := `{
     "parameters": [
         {
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
```

Comment:
This produce an error in swagger editor.
```
Structural error at paths ./api/something.get.parameters.0
should NOT have additional properties
additionalProperty: type, items
```
Did you want to make the PR more consistent?

Reply:
Sorry,maybe my gin-swagger is of very old version. I see there is no type displayed below the param name, so I assigned array type to its SimpleSchemaType. Have removed it. A new PR.

Reply:
@ubogdan thank you for point out.